ipython/fragment_reattachment_example.ipynb

```diff
@@ -0,0 +1,1102 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "for hd in hdl:\n",
+    "    bins = hd[1]\n",
+    "    counts = hd[0]\n",
+    "    plt.hist(bins[:-1], bins, weights=counts, alpha=0.1, color='blue')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b2c3d98a",
+   "metadata": {},
+   "source": [
+    "- List the molecules with the highest concentrations"
+   ]
+  },
+  {
```

Comment:
The output from this cell is missing, and there are a handful of empty cells after it that can be removed.